From ff4fe9cf99e315c254bbb35297e541e6c437b4ca Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 20:10:33 +0100
Subject: [PATCH 4/8] ia64: Add xattr *at family syscalls

setxattrat(), getxattrat(), listxattrat(), and removexattrat() were
added after ia64 was removed from the kernel.

Fixes: 6140be90ec70 ("fs/xattr: add *at family syscalls")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index f3e0ece43d34..88b694ac4540 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -383,3 +383,7 @@
 460	common	lsm_set_self_attr		sys_lsm_set_self_attr
 461	common	lsm_list_modules		sys_lsm_list_modules
 462	common	mseal				sys_mseal
+463	common	setxattrat			sys_setxattrat
+464	common	getxattrat			sys_getxattrat
+465	common	listxattrat			sys_listxattrat
+466	common	removexattrat			sys_removexattrat
-- 
2.52.0

